src/scalper.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
can these be more targeted than just disabling entire files.

Reply:
will plan to address linting later